web/src/data/songs_of_the_week.js

```diff
@@ -89,6 +89,66 @@ const songs = [
       spotify: "https://open.spotify.com/track/3V9cM3nCH2G66afoDi0snu",
     }
   ),
+  new SongOfTheWeek(
+    createDate("09/14/2024"),
+    null,
+    {
+      name: "Young",
+      artist: "Vallis Alps",
+      year: "2015",
+    },
+    {
+      albumCoverImage:
+        "https://i.scdn.co/image/ab67616d00001e021fb3115e3b35b94a49d6fe20",
+      youtube: "https://youtu.be/yoZPVMEsbeQ?si=bw2zzibpU-cYddpr",
+      spotify: "https://open.spotify.com/track/0ad87RTfHWmC2Syh1nOxEd",
+    }
+  ),
+  new SongOfTheWeek(
+    createDate("09/21/2024"),
+    null,
+    {
+      name: "Judy You Hung the Moon",
+      artist: "HARBOUR",
+      year: "2017",
+    },
+    {
+      albumCoverImage:
+        "https://i.scdn.co/image/ab67616d00001e02356628262c803315eece6380",
+      youtube: "https://youtu.be/psD04pozpcw?si=BhY5CnJxfjWYte9b",
+      spotify: "https://open.spotify.com/track/10oDDj46J0IMYBPSloeJ4Z",
+    }
+  ),
+  new SongOfTheWeek(
+    createDate("09/28/2024"),
+    null,
+    {
+      name: "Spooky",
```

Comment:
Do we want to save the Spooky song for October?

Reply:
good idea, lemme update